app/const.go

```diff
@@ -0,0 +1,8 @@
+package app
+
+const (
+	// Prefix of bech32 encoded address
+	AccountAddressPrefix = "cosmos"
+	// Application name
+	AppName = "sei-chain"
```

Comment:
we probably want to change this to "sei" instead of "sei-chain", so that the cmd dir will be the same name as the seid. See ticket 0030 and https://github.com/sei-protocol/sei-chain/pull/12